packages/cli/src/commands/client-task-queue/list.ts

```diff
@@ -53,12 +53,12 @@ export default class ClientTaskQueueList extends MeecoCommand {
     }
 
     const environment = await this.readEnvironmentFile();
-    const authConfig = await this.readConfigFromFile(AuthConfig, auth);
-    const service = new ClientTaskQueueService(environment, this.log);
-
+    let authConfig = (await this.readConfigFromFile(AuthConfig, auth))?.overrideWithFlags(flags);
```

Comment:
**Something to consider.**

Wouldn't passing `delegationId` as a separate optional argument to the config reader be a bit nicer to read? E.g.:
Instead of:
```
(await this.readConfigFromFile(AuthConfig, auth))?.overrideWithFlags(flags);
```
we could have
```
await this.readConfigFromFile(AuthConfig, auth, flags.delegationId)
```

Reply:
Yea I had a bit of a discussion with Zak about this for some reason we landed on this as a solution even though it's a bit ugly, I think because it was a bit more clear to someone reading the code what was going on there and the CLI's main purpose is for external developers to use it and read the code to see how the sdks work.

Reply:
Like when you look at that line it's pretty clear that some config is being overwritten, but if we simplify it the be an extra flag to be passed in you'd have to open up the readConfigFromFile source code to understand what it does.
Also I suppose if we ever need to override some other values with flags we can just shove them in there maybe.